torch/csrc/distributed/c10d/Ops.cpp

```diff
@@ -118,6 +118,26 @@ c10::intrusive_ptr<ProcessGroup::Work> barrier(
       BarrierOptions{device_ids, std::chrono::milliseconds(timeout)});
 }
 
+c10::intrusive_ptr<ProcessGroup::Work> send(
```

Comment:
trailing underscore?

Reply:
Send is not a in-place op. So there is no needs to have a trailing _.